packages/website-frontend/src/styles.scss

```diff
@@ -8,6 +9,7 @@ $theme-colors: (
   "danger": #dc3545
 );
 @import "~bootstrap/scss/bootstrap";
```

Comment:
This import can be removed if it is no longer used within the angular application

Reply:
Would probably shave off quite a few kB's .